src/encoded/batch_download.py

```diff
@@ -175,6 +177,21 @@ def get_peak_metadata_links(request):
     )
     return [peak_metadata_tsv_link, peak_metadata_json_link]
 
+
+def is_file_visualizable(file):
+    """Determines whether a file can be visualized in a genome browser.
+    Needs to be kept in sync with isFileVisualizable in objectutils.js.
+
+    Keyword arguments:
+    file -- file object including props to test for visualizability
+    """
+    return (
```

Comment:
Formatting nit:

```
def is_file_visualizable(file):
    """Determines whether a file can be visualized in a genome browser.
    Needs to be kept in sync with isFileVisualizable in objectutils.js.
    Keyword arguments:
    file -- file object including props to test for visualizability
    """
    conditions = [
        file.get('file_format') in [
            'bigWig',
            'bigBed',
        ],
        file.get('file_format_type') not in [
            'bedMethyl',
            'bedLogR',
            'idr_peak',
            'tss_peak',
            'pepMap',
            'modPepMap',
        ],
        file.get('status') in [
            'released',
            'in progress',
            'archived',
        ],
    ]
    return all(conditions)
```

Reply:
Isn't this code supposed to be in visualization.py?  Or some analog?  Not that I would recommend it unless you were really into "italian food"?

Reply:
Huh. I never thought of visualization.py as a kind of library you import utility functions from, but I guess that makes sense.

Reply:
probably visualization and batch_download and cart etc. should all import a vis module.  Perhaps this is vis_defines.py?